src/leo-client-app/src/components/SatelliteInfo.tsx

```diff
@@ -44,13 +45,13 @@ const SatelliteInfo: React.FC = () => {
   });
 
   const fetchData = () => {
-    fetch('http://localhost:3001/getSatelliteInfo') // TODO: Fix endpoint for when server is hosted
+    fetch("http://localhost:3001/getSatelliteInfo") // TODO: Fix endpoint for when server is hosted
```

Comment:
u funny for this change

Reply:
lol i think that's my prettier acting up